docs/user/gcp/quota.md

```diff
@@ -0,0 +1,13 @@
+# Handling quota on GCP
+
+All cloud providers have some concept of limits imposed to protect their services from misuse as well as protect customers from mistakes that can leads to excessive charges.  In GCP these will vary by region, project and account.  This is great news in terms of flexibility but can present an initial hurdle if the defaults in your project are too low.
+
+## Common problems
+The most likely areas of quota contention are for CPU, Static IPs and Storage (SSD).  A vanilla IPI installation will result in 24 CPUs, 3 Static IPs and 768 GB of storage consumed.  To be clear, that's the absolute bare minimum.  It's critical to leave enough headroom for growth especially for projects shared by multiple teams.  Whenever an installation failes it's best to check `.openshift_install.log` for errors.
+
+~~~
+Error: Error waiting for instance to create: Quota 'SSD_TOTAL_GB' exceeded.  Limit: 500.0 in region europe-west1. 
+~~~
+
+## Making adjustments
+  To adjust quotas visit the [GCP console](https://console.cloud.google.com/iam-admin/quotas) and make necessary changes.  This will likely involve filing a support ticket so it's best to plan ahead as this is often the most time consuming barrier to your first running cluster.  For more detailed information please refer to the [GCP documentation](https://cloud.google.com/compute/quotas).
```

Comment:
can you collect all the links towards the end, like https://github.com/openshift/installer/blame/master/docs/user/azure/limits.md#L125

Reply:
Sure.  I'll follow the structure of that doc.